cmd/pbm/main.go

```diff
@@ -244,6 +244,10 @@ func main() {
 		StringVar(&restore.pitrBase)
 	restoreCmd.Flag("num-parallel-collections", "Number of parallel collections").
 		Int32Var(&restore.numParallelColls)
+	restoreCmd.Flag("num-insertion-workers-per-collection",
+		"Specifies the number of insertion workers to run concurrently per collection. For large imports, "+
+			"increasing the number of insertion workers may increase the speed of the import.").
+		Int32Var(&restore.numInsertionWorkers)
```

Comment:
```suggestion
		Default("1").
		Int32Var(&restore.numInsertionWorkers)
```

set default value. in `pbm help restore` it will show `--num-insertion-workers-per-collection=1`